ParkingLot/src/com/parkingLot/models/vehicles/Vehicle.java

```diff
@@ -0,0 +1,53 @@
+package com.parkingLot.models.vehicles;
+
+public class Vehicle {
+    boolean isParked;
+    String vehicleType;
+    String vehicleRegistrationNumber;
+    String vehicleColor;
+    String vehicleTicketId;
```

Comment:
Since this is the Vehicle class the vehicle prefix is unnecessary in it's properties.

Reply:
Done changes. Removed unnecessary "vehicle" prefix.
[#change](https://github.com/workattech/mock-machine-coding-1/pull/90/commits/af23a8684b7cb2d1147eb97f7a0be71146622908)